vuex-options/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@kodadot1/vuex-options",
-  "version": "0.0.1-rc.7",
+  "version": "0.0.1-rc.8",
```

Comment:
```suggestion
  "version": "0.0.1-rc.9",
```

Reply:
Actually, I update 0.0.1-rc.7 to 0.0.1-rc.8 for my commit.
So 0.0.1-rc.8 to 0.0.1-rc.9 update nothing 😂